code-review.md

```diff
@@ -0,0 +1,43 @@
+# Implementing a Strong Code Review Culture
+
+## Abstract
+
+Code review is about catching bugs, right? Wrong. Modern code review is about
+socialization, learning, and teaching.
+
+What's the true value of code review? How can I get the most out of a peer's
+review and how can I review code without being seen as overly critical? Code
+review is a skill that will make you a better developer and a better teammate.
+
+You will leave this talk with the tools to implement a successful code review
+culture. You'll learn how to get more from the reviews your already getting and
```

Comment:
You'll learn how to get more from the reviews your already getting and

to

You'll learn how to get more from the reviews you're already getting and